README.md

```diff
@@ -107,6 +107,12 @@ Object of scan objects containing the following parameters
 
 See ["Scanning MDX Content"](#scanning-mdx-content) for more details.
 
+### onContent
+
+> `function` | optional
+
+A function that will run on build for each MDX page. All metadata and full text content are passed to this function as its only argument. Useful for indexing your content for site search or any other purpose where you'd like to capture content on build.
+
```

Comment:
Totally possible I'm not capturing this well but I think it boils down to the above.  Any ideas on how we should word it / pitch it let me know. 

One thing that comes to mind is I currently do not have a `use-cases.mdx` which I did for the `scan` option.. Might not be worth it though. Any feedback is appreciated here.